app.js

```diff
@@ -0,0 +1,35 @@
+const navLinks = document.querySelector('.nav-links');
+const hamburger = document.getElementById('open-menu');
+const closeMenu = document.getElementById('close-menu');
+
+const navLink1 = document.getElementById('nav-link1');
+const navLink2 = document.getElementById('nav-link2');
+const navLink3 = document.getElementById('nav-link3');
+
+const wrap = document.querySelector('.wrapper');
+
+// listen for individual clicks
+navLink1.addEventListener('click', () => {
+  navLinks.classList.toggle('active');
+  wrap.classList.toggle('w-active');
+});
+navLink2.addEventListener('click', () => {
+  navLinks.classList.toggle('active');
+  wrap.classList.toggle('w-active');
+});
+navLink3.addEventListener('click', () => {
+  navLinks.classList.toggle('active');
+  wrap.classList.toggle('w-active');
+});
+
```

Comment:
**[OPTIONAL]**

- Hey @Team, great job with the mobile menu. Your implementation works fine however, the codebase is not DRY as many codes are being repeated as this is not a good practice. This could be simplified further below:
 
- You could use the `querySelectorAll` on the `nav-links` to select all the links so you could utilize the `forEach` method so you could loop through all the links instead of giving them different names like this:

``` 
// const navLink1 = document.getElementById('nav-link1');
const navLink = document.querySelectorAll('.nav-link'); // 
// const navLink2 = document.getElementById('nav-link2'); 
// const navLink3 = document.getElementById('nav-link3');
```

**Update your index.html FROM**

``` 
<ul>                     
     <li><a id="nav-link1" href="#Portfolio">Portfolio</a></li>
     <li><a id="nav-link2" href="#About">About</a></li>
      <li><a id="nav-link3" href="#Contact">Contact</a></li>
</ul>
```


**Update your index.html TO**

``` 
<ul>                     
    <li><a class="nav-link" href="#Portfolio">Portfolio</a></li>
    <li><a class="nav-link" href="#About">About</a></li>
    <li><a class="nav-link" href="#Contact">Contact</a></li>
</ul>
```

**UPDATE the js To**

```

// This will loop over each nav-link when been clicked
navLink.forEach((link) => {
  link.addEventListener('click', () => {
    navLinks.classList.toggle('active');
    wrap.classList.toggle('w-active');
  });
});

 // listen for individual clicks
// navLink1.addEventListener('click', () => {
//   navLinks.classList.toggle('active');
//   wrap.classList.toggle('w-active');
// });
// navLink2.addEventListener('click', () => {
//   navLinks.classList.toggle('active');
//   wrap.classList.toggle('w-active');
// });
// navLink3.addEventListener('click', () => {
//   navLinks.classList.toggle('active');
//   wrap.classList.toggle('w-active');
// });

```
**NOTE: I left the commented code so you can see the changes well enough. You shouldn't add commented code in your codebase as this will make your codebase look dirty and not appear professional**

Reply:
@ziecco I tried doing that but the javascript eslint linter did not allow me to use the for loop any suggestions

Reply:
@OkoyeCharles, I actually have your repo working with no linter error from my machine. I could only be of help if you explain the linter error more in detail

Reply:
@zieeco thank you very much for the descriptive explanations... It worked for me, I used the for let... loop before but now the forEach loop doesn't have any problems.